### ISSUE TITLE
wearos app

### ISSUE THREAD

**Issue by simonfossom** [adrcotfas/Goodtime#343]
For anyone wanting distraction free environment wearos app would be invaluable. just leave the phone somewhere else and do the task.

**Comment by hojjatabdollahi** [comment]
Yeah. I just want my watch to tell me what's up and I only need "+1 minute", "start break" on it. But I want it to capture my attention, happens a lot that I go over my break time.
